backend/manager/modules/vdsbroker/src/test/java/org/ovirt/engine/core/vdsbroker/monitoring/VirtMonitoringStrategyTest.java

```diff
@@ -69,6 +81,15 @@ public void testVirtCanMoveToMaintenance() {
         assertTrue(virtStrategy.canMoveToMaintenance(vds));
         doReturn(true).when(virtStrategy).isAnyNonExternalVmRunningOnVds(vds);
         assertFalse(virtStrategy.canMoveToMaintenance(vds));
+        vds.setId(vdsId3);
+        doReturn(false).when(virtStrategy).isAnyNonExternalVmRunningOnVds(vds);
+        assertTrue(virtStrategy.canMoveToMaintenance(vds));
+        vds.setId(vdsId4);
+        doReturn(false).when(virtStrategy).isAnyNonExternalVmRunningOnVds(vds);
+        assertTrue(virtStrategy.canMoveToMaintenance(vds));
+        vds.setId(vdsId5);
+        doReturn(true).when(virtStrategy).isAnyNonExternalVmRunningOnVds(vds);
```

Comment:
why do we need that?

Reply:
It tests all the tests that @barpavel wrote above

Reply:
right but I mean - why do we need isAnyNonExternalVmRunningOnVds to return false here? I'd expect either:
(1) not to mock it since we don't expect it to be called; or
(2) return false to make sure we fail due to a different reason

Reply:
done changing @ahadas please review